app/scripts/lib/transaction/metrics.ts

```diff
@@ -848,6 +848,7 @@ async function buildEventFragmentProperties({
       TransactionType.tokenMethodSetApprovalForAll,
       TransactionType.tokenMethodTransfer,
       TransactionType.tokenMethodTransferFrom,
+      TransactionType.tokenMethodIncreaseAllowance,
```

Comment:
[nit] could move up to maintain alphabetical order. same comment to other instances below